Fix "ordered comparison between pointer and zero ('char *' and 'int')"
https://sourceforge.net/p/mimetic/bugs/2/
--- mimetic/os/mmfile.cxx.orig	2014-06-17 03:12:00.000000000 -0500
+++ mimetic/os/mmfile.cxx	2018-11-01 03:53:21.000000000 -0500
@@ -57,7 +57,7 @@
 bool MMFile::map()
 {
     m_beg = (char*) mmap(0, m_st.st_size, PROT_READ, MAP_SHARED,m_fd,0);
-    if(m_beg > 0)
+    if(m_beg != NULL)
     {
         m_end = m_beg + m_st.st_size;
         #if HAVE_MADVISE
